prbt_hardware_support/spec.dox

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2019 Pilz GmbH & Co. KG
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @page Spec
+ *
+ *  @tableofcontents
+ *  @section system_specification System Specification
+ *  @brief System Specification
+ *
+ *  @subsection ISO-10218-1
+ *
+ *
+ *
+ * @subsubsection sto_stop1 Safe torque off (STO) and  Stop 1
+ *
+ * @definereq{Stop1_Trigger}
+ *  - A Stop 1 is triggered, if at least one of the following cases is true:
+ *    - the state of the STO changes from TRUE -> FALSE,
+ *    - connection to safety controller is lost,
+ *    - the messages informing about the STO (coming from the
+ *      safety controller) is incomplete or in any way corrupted,
+ *    - the communication protocol used by the safety controller
+ *      does not fulfill the expected specification.
+ *
+ * @definereq{No_new_commands_during_STO_false}
+ *  - As long as the STO is FALSE, no new motions commands are allowed.
+ *
+ * @definereq{Hold_driver_if_STO_false}
+ *  - If the STO changes from TRUE -> FALSE the driver is halt.
+ *
+ * @definereq{Recover_driver_after_STO_false}
+ *  - If the STO changes from FALSE -> TRUE, the driver is recovered.
```

Comment:
Should this also state the unholding?

Reply:
Indirectly covered requirement "No_new_commands_during_STO_false"